docs/eog/case-studies-cw20-timeline.rst

```diff
@@ -0,0 +1,145 @@
+.. _CW20-Timeline: 
+
+CW20 Timeline
+=================
+
+We made the decision three weeks before CW20 to move the event online. 
+We had less than one month to contact all of our stakeholders (venue, accommodation, sponsors, speakers, workshop facilitators and of course participants), inform them of the decision and renegotiate everything that had been previously agreed. 
+As days passed and the implications around the global pandemic became more clear, stakeholders were very understanding and accommodating. 
+Below is a breakdown of what we did. 
+
+Event Project Stage
```

Comment:
The Event Project Stage is a container - it covers the stage when you get the big 'OK' to move forward with your event - so Pre-Event, During the Event and Post-Event are all contained in the Event Project Stage - so it's better to  put sub-headings in to reflect the sub-stages within the Event Project Stage.

Reply:
I have just removed those other headings, as heading levels in restructuredtext are not assigned to certain characters but the structure is determined from the succession of headings.